registration-processor/registration-processor-packet-manager/pom.xml

```diff
@@ -7,12 +7,12 @@
 	<parent>
 		<groupId>io.mosip.registrationprocessor</groupId>
 		<artifactId>registration-processor</artifactId>
-		<version>1.1.5.5</version>
+		<version>1.1.5.5-p1-SNAPSHOT</version>
 	</parent>
 
 	<artifactId>registration-processor-packet-manager</artifactId>
 	<name>registration-processor-packet-manager</name>
-	<version>1.1.5.5</version>
+	<version>1.1.5.5-p1-SNAPSHOT</version>
```

Comment:
Lets not change this version, also we should ensure this is not getting built

Reply:
done

Reply:
done